src/main/java/nextstep/fp/Conditional.java

```diff
@@ -0,0 +1,6 @@
+package nextstep.fp;
+
+public interface Conditional {
+
+    boolean test(Integer number);
```

Comment:
```suggestion

@FunctionalInterface
public interface Conditional {

    boolean test(Integer number);
```
함수형 인터페이스로 사용한다면 애너테이션을 붙여주는 것이 좋습니다.

<img width="496" alt="image" src="https://github.com/next-step/java-ladder/assets/36617783/8661d83f-d1ff-475b-bf32-dc572b0b1ef7">

정확히 명시할 수 있고 실수로 메서드가 추가되면 IDEA에서 경고가 나타나요